src/RemoteRequest/CachedRemoteGetRequest.php

```diff
@@ -101,42 +102,48 @@ public function __construct(
 	 *
 	 * @param string $url URL to get.
 	 * @return Response Response for the executed request.
-	 * @throws FailedToGetFromRemoteUrl If retrieving the contents from the URL failed.
+	 * @throws FailedToGetFromRemoteUrl|FailedToGetCachedResponseData If retrieving the contents from the reponse failed.
```

Comment:
Actually, the `FailedToGetFromRemoteUrl` is being caught, so the only one that might be thrown here is `FailedToGetCachedResponseData`.

However, that actually contradicts the interface `RemoteGetRequest`... So we should either adapt the interface to consider both types of exceptions, or roll everything into one agaon.